backend/backend/views.py

```diff
@@ -67,15 +68,14 @@ def get(self, request, *args, **kwargs):
             BearerTokenSerializer(token).data
             for token in BearerToken.objects.filter(user=request.user).order_by("-created")
         ]
-        try:
-            implicit_token = ImplicitBearerTokenSerializer(ImplicitBearerToken.objects.get(user=request.user)).data
-
-        except ObjectDoesNotExist:
-            implicit_token = None
+        implicit_tokens = [
+            ImplicitBearerTokenSerializer(token).data
+            for token in ImplicitBearerToken.objects.filter(user=request.user).order_by("-created")
+        ]
         return ApiResponse(
             {
                 "tokens": tokens,
-                "implicit_token": implicit_token,
+                "implicit_tokens": implicit_tokens,
```

Comment:
This endpoint used to return a token or None, now it's always a list (potentially empty). Is this endpoint used externally, in which case we need to adjust?

Reply:
Good observation. I'm not aware of anything using this at the moment, the plan was to make these tokens visible in the frontend (following remarks from the audit) but it has yet to be implemented.